public/help/help.md

```diff
@@ -27,5 +27,6 @@ The following procedure describes adding and updating settings of custom tracks.
    * A start position and a distance (e.g. "chr1:1+100")
    * A node ID anchor and a distance (e.g. "node:100+10")
 ![Region Input Options](helpGuideImages/img8.png)
-9. Click Go to see the selected tracks render in the visualization area.
+4. There is an option to click on the "Simplify Off" button to enable vg simplify, which would remove eliding small snarls in the graph. This would only work for BED files or graph tracks (not reads or haplotypes). 
```

Comment:
This makes it sound like you can turn on simplification for all BED file chunks. It also reads like the user should expect to see a non-functional simplify button sometimes, when actually the button only appears when it is going to work.

I think we want to communicate that the button is not always visible (so people aren't surprised when they don't see it), and that people should expect to see it only when there aren't any reads to be displayed.

I think it *does* work with haplotype tracks; the haplotype information goes into `vg chunk` and the `vg simplify` pass happens after that.